### PR TITLE
Projection + update model of messages

### PR DESCRIPTION
**Pull request by jkralik**

### CODE REVIEW

File: uri/uri.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package uri
+
+// Authorization Service URIs.
```
Comment (rafajpet) [inline uri/uri.go]: It's Resource Service URIs :)

File: protobuf/resources/events/events.proto [hunk 2]
```diff
@@ -9,55 +9,62 @@ option go_package = "github.com/go-ocf/resources/protobuf/resources/events;event
...
 // https://github.com/openconnectivityfoundation/core/blob/master/oic.wk.rd.raml#L106
 message ResourcePublished {
-    string id = 1;
-    ocf.cloud.resources.Resource resource = 2; // Resource publish is atomic. The CoAP-Gateway is responsible for the transaction.
-    string published_by = 3;
-    int32 time_to_live = 4;
-    ocf.cloud.resources.AuditContext audit_context = 5;
-    ocf.cloud.resources.EventMetadata event_metadata = 6;
+    string id = 1; // calucalated from href as uuid.NewV5(uuid.NamespaceURL, href)
+    string href = 2;
```
Comment (ondrejtomcik) [inline protobuf/resources/events/events.proto]: Why do we need href as it is in the resource itself, which is part of the event?

File: protobuf/resources/events/events.proto [hunk 3]
```diff
@@ -9,55 +9,62 @@ option go_package = "github.com/go-ocf/resources/protobuf/resources/events;event
...
 }
 
 // https://github.com/openconnectivityfoundation/core/blob/master/oic.wk.rd.raml #Specification CR needed
 message ResourceUnpublished {
-    string id = 1;
-    string unpublished_by = 2;
-    ocf.cloud.resources.AuditContext audit_context = 3;
-    ocf.cloud.resources.EventMetadata event_metadata = 4;
+    string id = 1; // calucalated from href as uuid.NewV5(uuid.NamespaceURL, href)
+    string href = 2;
```
Comment (ondrejtomcik) [inline protobuf/resources/events/events.proto]: Same here. Here we don't have resource msg, but do we need href? Also on other places.

Comment (jkralik) [reply protobuf/resources/events/events.proto]: Right it s not need fo events -> because we can determine href from projection (event ResourcePublished)

**Pull request merged**